vmdb/product/toolbars/ems_infra_center_tb.yaml

```diff
@@ -22,6 +22,11 @@
       :text: 'Edit this #{ui_lookup(:table=>"ems_infra")}'
       :title: 'Edit this #{ui_lookup(:table=>"ems_infra")}'
       :url: '/edit'
+    - :button: ems_infra_scale
```

Comment:
@Ladas : the button show/hide enable/disable conditions are hard-coded in the application_helper `build_toolbar_hide_button` and friends :-(

Technical debt. Any volunteer to fix it?